test/functional/fixtures/regression/gh-1057/testcafe-fixtures/index-test.js

```diff
@@ -0,0 +1,21 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `gh-1057`
+    .page `http://localhost:3000/fixtures/regression/gh-1057/pages/index.html`;
+
+const targetsClicked = ClientFunction(() => window.target1Clicked && window.target2Clicked);
+
+test('gh-1057', async t => {
+    // NOTE: scrolling has issues in iOS Simulator https://github.com/DevExpress/testcafe/issues/1237
+    await t
+        .wait(5000)
```

Comment:
And I don't get what does this test do. Why we need wait 5000 at start of the test, then hover `target2`, then wait and hover `target2` again? If it's all for ios let's skip it for this test